README.md

```diff
@@ -76,6 +76,12 @@ Other modules:
     When to use this module: you are already using Java Platform/System Logger in some other project, and you like to have same log outputs.
     See [Java Platform/System Logger](docs/jpl.md) section for more details.
 
+* java.util.logging bridge - with this logging bridge, it is possible to use `zio-logging` for JUL loggers (usually third-party non-ZIO libraries), add the one of following lines to your `build.sbt` file:
+
+    ```scala
+    // JUL bridge
+    libraryDependencies += "dev.zio" %% "zio-logging-jul-bridge" % "2.2.2"
+    ```
```

Comment:
is good to add something like

```
See [java.util.logging bridge](docs/jul-bridge.md) section for more details.
```

for quick navigation

Reply:
Fixed